website/docs/migrate-v3-to-v4.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: migrate-v3-to-v4
+title: Migrating to v4
+---
```

Comment:
ok good to go on my end if you are @MateusAndrade  👍 

<img width="1566" alt="Screen Shot 2020-09-28 at 1 44 59 PM" src="https://user-images.githubusercontent.com/1047502/94467823-79c87c80-0191-11eb-89d2-afd0b8b78bb0.png">

Reply:
Nice!!!!! Thanks a lot, @sterlingwes! 🚀 🚀 🚀 🚀 🚀 🚀